.travis.yml

```diff
@@ -0,0 +1,34 @@
+sudo: false
+dist: trusty
+language: node_js
+
+# Support Active LTS versions of Node.js
+node_js:
+  - '8'
```

Comment:
I guess we don't need the version `8` here. @shumkov What do you think?

Reply:
Why? It's still on maintenance https://nodejs.org/en/about/releases/